src/api/Jalasoft.TeamUp.Projects.API/Controllers/DeleteProjectById.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace Jalasoft.TeamUp.Projects.API.Controllers
+{
+    using System;
+    using System.Net;
+    using Jalasoft.TeamUp.Projects.Core.Interfaces;
+    using Jalasoft.TeamUp.Projects.Models;
+    using Jalasoft.TeamUp.Projects.ProjectsException;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.Http;
+    using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+    using Microsoft.OpenApi.Models;
+
+    public class DeleteProjectById
+    {
+        private readonly IProjectsService projectsService;
+
+        public DeleteProjectById(IProjectsService projectsService)
+        {
+            this.projectsService = projectsService;
+        }
+
+        [FunctionName("DeleteProjectById")]
+        [OpenApiOperation(operationId: "DeleteProjectById", tags: new[] { "Projects" })]
+        [OpenApiParameter(name: "id", In = ParameterLocation.Path, Required = true, Type = typeof(Guid), Description = "The project identifier.")]
+        [OpenApiResponseWithoutBody(statusCode: HttpStatusCode.NoContent, Description = "There's no content to be returned")]
```

Comment:
is this status code part of the possible scenarios? maybe yes if the user sends an invalid id that is not found in database right? but if it gets deleted what http response should be used?